postman/Get Broadcast Stats.postman_collection.json

```diff
@@ -0,0 +1,482 @@
+							"  if (pm.expect(jsonData.broadcast_id).to.be.above(0)) {",
+							"      pm.globals.set(\"broadcastId\", jsonData.broadcast_id);",
+							"  }",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{broadcast_self_link}}",
+					"host": [
+						"{{broadcast_self_link}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Stats Entries",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "c51162ab-dfa9-42e5-82da-3acf6ca35bd1",
+						"exec": [
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/campaigns/b{{broadcastId}}/stats",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"campaigns",
+						"b{{broadcastId}}",
+						"stats"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Clicks by Link",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/campaigns/b{{broadcastId}}/stats/clicks_by_link",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"campaigns",
+						"b{{broadcastId}}",
+						"stats",
+						"clicks_by_link"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Daily Unique and Total Clicks",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/campaigns/b{{broadcastId}}/stats/daily_clicks",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"campaigns",
+						"b{{broadcastId}}",
+						"stats",
+						"daily_clicks"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Daily Unique and Total Opens",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": ""
+				},
+				"url": {
+					"raw": "{{base_url}}/accounts/{{accountId}}/lists/{{listId}}/campaigns/b{{broadcastId}}/stats/daily_opens",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"accounts",
+						"{{accountId}}",
+						"lists",
+						"{{listId}}",
+						"campaigns",
+						"b{{broadcastId}}",
+						"stats",
+						"daily_opens"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Get Daily Sales",
```

Comment:
I got a 404 on this, probably because my broadcast didn't have any sales. Is there a way to tell postman that a 404 is okay in this case?

Reply:
Moved response code check to request level tests to accommodate this.